### DISCUSSION TITLE
Saving settings

### DISCUSSION THREAD

**Discussion by mcp1766** [phcode-dev/phoenix#1194] [Q&A]
Hey,

Just try phcode.  Installed some themes and extensions.  Then restart browser and all set to default (no extra themes ans extensions).
Can we save them for later use ?
Thk.

**Comment by abose** [comment]
Hi @mcp1766 which browser is this?

**Reply by mcp1766** [reply]
Google Chrome.  I'am under Manjaro KDE  23.04 .  Installing with  yay -S google-chrome.

**Reply by abose** [reply]
It should save and restore the settings under google chrome. 

Any chance you were using two tabs of phcode at the same time? We have a bug where when there are two tabs settings change like theme switch etc are not saved.

**Reply by mcp1766** [reply]
Just open in only one tab in Chrome.  The settings are gone when I close completely Chrome then restart it and launch the app again.  Then everything are gone.  If a quit the app but Chrome still active then reopen the app (with help of a bookmark) then it is ok but not if a close all tabs and Chrome.

**Reply by abose** [reply]
Quite strange behavior. Need a bit more context to solve this.
1. Is any file contents lost? 
2. Are all installed extensions lost after chrome restart.
3. Which OS, is there any corporate firewall or anything else peculiar

I can check it out over google meet or any remote desktop session if you want it addressed. 

Alternatively any screen recordings will also help with the problem

**Reply by mcp1766** [reply]
My apologies, I discovered phoenix by going directly to https://phcode.dev/ from a Google search. I don't notice since there is a local procedure with npm. Now it's ok.
Another question, a number of extensions cannot be installed. I saw that the directory is located in phoenix/src/extensions/default. Can we install them manually with git clone in this directory and load them? THANKS

**Reply by abose** [reply]
Can you give the names of extension that failed to install? Is it from the extension store? Any screen shot of the error message would help too.
